guest-os-info.md

```diff
@@ -0,0 +1,34 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``kubevirt.io/os`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    kubevirt.io/os: win2k12r2
+```
+
+The ``kubevirt.io/os`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/)
+database.
+The following Short IDs are currently supported:
+
+| Short ID | Name | Version | Family | ID |
+| --- | --- | --- | --- | --- |
+| **fedora26** | Fedora 26 | 26 | linux | http://fedoraproject.org/fedora/26 |
+| **fedora27** | Fedora 27 | 27 | linux | http://fedoraproject.org/fedora/27 |
+| **rhel7.0** | Red Hat Enterprise Linux 7.0 | 7.0 | linux | http://redhat.com/rhel/7.0 |
+| **rhel7.1** | Red Hat Enterprise Linux 7.1 | 7.1 | linux | http://redhat.com/rhel/7.1 |
+| **rhel7.2** | Red Hat Enterprise Linux 7.2 | 7.2 | linux | http://redhat.com/rhel/7.2 |
+| **rhel7.3** | Red Hat Enterprise Linux 7.3 | 7.3 | linux | http://redhat.com/rhel/7.3 |
+| **rhel7.4** | Red Hat Enterprise Linux 7.4 | 7.4 | linux | http://redhat.com/rhel/7.4 |
+| **win2k12r2** | Microsoft Windows Server 2012 R2 | 6.3 | winnt | http://microsoft.com/win/2k12r2 |
```

Comment:
As far as I can see, win2k12r2 is the only one we currently have a PR for.

Reply:
>As far as I can see, win2k12r2 is the only one we currently have a PR for.

we have this table under developer documents. so i can narrow this doc only for the existing PR and maintain it as we go along with new PRs.

Reply:
Yes please. :+1: 
As a general rule, we only document what we have, not what we plan to do.